docs/examples/custom-json/README.md

```diff
@@ -124,30 +124,35 @@ For that, we can use the Python types that `mlserver` provides out of box, or we
 ```python
 import requests
 import json
+from pprint import PrettyPrinter
 
-inputs = {
-    "name": "Foo Bar",
-    "message": "Hello from Client (REST)!"
-}
+pp = PrettyPrinter(indent=1)
+
+inputs = {"name": "Foo Bar", "message": "Hello from Client (REST)!"}
 
 # NOTE: this uses characters rather than encoded bytes. It is recommended that you use the `mlserver` types to assist in the correct encoding.
-inputs_string= json.dumps(inputs)
+inputs_string = json.dumps(inputs)
 
 inference_request = {
     "inputs": [
         {
             "name": "echo_request",
             "shape": [len(inputs_string)],
             "datatype": "BYTES",
-            "data": [inputs_string]
+            "data": [inputs_string],
         }
     ]
 }
 
 endpoint = "http://localhost:8080/v2/models/json-hello-world/infer"
 response = requests.post(endpoint, json=inference_request)
 
-response.json()
+print(f"full response:\n")
+print(response)
+# retrive text output as dictionary
+output = json.loads(response.outputs[0].contents.bytes_contents[0])
```

Comment:
These look like the fields for gRPC outputs. However, they will be different in the case of REST.

To simplify things, could you use the `StringRequestCodec` instead? That would look something like this (I'm writing this from the top of my head, so pls double check the imports and methods):

```python
import json

from mlserver.types import InferenceResponse
from mlserver.codecs.string import StringRequestCodec

inference_response = InferenceResponse.parse_raw(response.text)
raw_json = StringRequestCodec.decode_response(inference_response)
output = json.loads(raw_json)
```

Reply:
Good spot! It will return a list of strings by default.

Reply:
Thanks! All done!